examples/python/observability/observability_greeter_client.py

```diff
@@ -47,16 +47,18 @@ def run():
     provider = MeterProvider(metric_readers=[reader])
     otel_plugin = BaseOpenTelemetryPlugin(provider)
 
-    with grpc_observability.OpenTelemetryObservability(plugins=[otel_plugin]):
-        with grpc.insecure_channel(target="localhost:50051") as channel:
-            stub = helloworld_pb2_grpc.GreeterStub(channel)
-            try:
-                response = stub.SayHello(
-                    helloworld_pb2.HelloRequest(name="You")
-                )
-                print(f"Greeter client received: {response.message}")
-            except grpc.RpcError as rpc_error:
-                print("Call failed with code: ", rpc_error.code())
+    grpc_observability.start_open_telemetry_observability(plugins=[otel_plugin])
+    # You can also use context manager to start observability:
+    # with grpc_observability.OpenTelemetryObservability(plugins=[otel_plugin]):
```

Comment:
As a rule of thumb, people will only ever use the version of an API that you include in your example. So you might want to ask yourself:

- Should the context manager or the global call be the default version?
- Should we include a separate example that includes the other option?

I'll let you make the call on both of those questions.

Reply:
One of the reasons why we added global init option is to keep in sync with other languages, since none of them have context manager option, I'll keep global call here as default version.